docs/html/cli/pip_wheel.rst

```diff
@@ -30,6 +30,13 @@ Description
 
 This is now covered in :doc:`../reference/build-system/index`.
 
+Differences to `build`
+----------------------
+
+`build <https://pypi.org/project/build/>`_ is a simple tool which can among other things build
+wheels for projects using PEP 517. It is comparable to the execution of ``pip wheel --no-deps .``.
+``pip wheel`` covers the wheel scope of ``build`` but offers many additional features.
```

Comment:
You should also mention that ``build`` also offers additional features; most importantly, it builds source distributions (pip does not). I would say _using `build` to build a wheel_ is comparable to _running `pip wheel --no-deps`_, but that’s where comparison of the tools end.